src/strategies/cyberkongz-v3/index.ts

```diff
@@ -0,0 +1,71 @@
+import { formatUnits } from '@ethersproject/units';
+import {getProvider, getSnapshots, multicall} from '../../utils';
+
+export const author = 'maxbrand99';
+export const version = '1.0.0';
+
+const bananaContract = '0xe2311ae37502105b442bbef831e9b53c5d2e9b3b';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)',
+  'function totalSupply() external view returns (uint256)'
+];
+
+export async function strategy(space, network, provider, addresses, options, snapshot) {
+  const allAddresses = {};
+  const promises: any = [];
+  const blocks = await getSnapshots(network, snapshot, provider, options.chains.map((s) => s.network || network));
+  const allCalls: any[] = [];
+  options.chains.forEach((chain) => {
+    const calls: any[] = [];
+    Object.keys(chain.registries).forEach((registry) => {
+      allAddresses[registry] = chain.registries[registry]
+      addresses.forEach((address: any) => {
+        calls.push([registry, 'balanceOf', [address]]);
+        allCalls.push([registry, 'balanceOf', [address]]);
+      });
+    });
```

Comment:
May concern is this, someone may pass multiple chains and registries here, and it may eat up some memory.